client/src/app/discover/keyword-search/keyword-search.js

```diff
@@ -14,40 +13,30 @@ function KeywordSearch() {
 
   const {
     recipes,
-    randomRecipes,
     hasResult,
     loading,
   } = useApi({ url });
 
+  useEffect(() => {
+    setUrl(`https://api.spoonacular.com/recipes/complexSearch?apiKey=${apiKey}&query=`);
```

Comment:
I see no point in `usingEffect` to set the initial value of the url.
The same could be achieved by changing line 12 to:
`const [url, setUrl] = useState(`https://api.spoonacular.com/recipes/complexSearch?apiKey=${apiKey}&query=`);`

Reply:
fixed